common/notifications.cpp

```diff
@@ -980,4 +982,64 @@ namespace rs2
 
         pinned = true;
     }
+
+    metadata_warning_model::metadata_warning_model()
+        : process_notification_model(nullptr)
+    {
+        enable_expand = false;
+        enable_dismiss = true;
+        update_state = 0;
+        pinned = true;
+        message = "Frame Metadata is device feature allowing\n"
+                  "software synchronization between different\n"
+                  "camera streams.\n"
+                  "It must be explicitly enabled on Windows OS\n";
+        last_progress_time = system_clock::now();
```

Comment:
Is draw_progress_bar() called at all? Where is this last_progress_time being used? If not used, is the inheritance from process_notification_model needed? Maybe inherit directly from notification_model?